buffer/buffer.mbt

```diff
@@ -84,11 +84,12 @@ pub fn to_string(self : T) -> String {
 /// Return a new unchecked string contains the data in buffer.
 /// Note this function does not validate the encoding of the byte sequence, 
 /// it simply copy the bytes into a new String.
-pub fn to_unchecked_string(self : T) -> String {
-  Bytes::from_fixedarray(self.data).to_unchecked_string(
-    offset=0,
-    length=self.len,
-  )
+pub fn to_unchecked_string(
+  self : T,
+  offset~ : Int = 0,
+  length~ : Int = self.len
```

Comment:
We may need to avoid labelled dependency: https://github.com/moonbitlang/core/pull/1231#discussion_r1849374941